Packs/MicrosoftExchangeOnline/pack_metadata.json

```diff
@@ -2,7 +2,7 @@
     "name": "Microsoft Exchange Online",
     "description": "Exchange Online and Office 365 (mail)",
     "support": "xsoar",
-    "currentVersion": "1.3.7",
+    "currentVersion": "1.4.0",
```

Comment:
The version bump should be a minor one and not a major one, change it to `1.3.8` instead of `1.4.0`.

Reply:
Done